pkg/docs/reference.md

```diff
@@ -145,6 +146,20 @@ Creates a tar file from a list of inputs.
         </p>
       </td>
     </tr>
+    <tr>
+      <td><code>compressor</code></td>
+      <td>
+        <code>Label, optional</code>
+        <p>Executable to be used as compression program.</p>
```

Comment:
maybe   program => filter, so it is obvious that the program does not specify input or output files.